packages/headless/src/controllers/insight/facets/category-facet/headless-insight-category-facet.test.ts

```diff
@@ -28,18 +24,22 @@ import {
   fetchFacetValues,
 } from '../../../../features/insight-search/insight-search-actions';
 import {updateFacetOptions} from '../../../../features/facet-options/facet-options-actions';
-import {CategoryFacetOptions} from '../../../core/facets/category-facet/headless-core-category-facet';
+import {
+  CategoryFacet,
+  CategoryFacetOptions,
+} from '../../../core/facets/category-facet/headless-core-category-facet';
```

Comment:
I think if you go through the trouble of re-exporting the `CategoryFacet` and `CategoryFacetOptions` in the `packages/headless/src/controllers/insight/facets/category-facet/headless-insight-category-facet.ts` file, you should import from this file instead of importing those from the core files no?

Reply:
```suggestion
import {
  CategoryFacet,
  CategoryFacetOptions,
} from './headless-insight-category-facet';
```